amgut/db/patches/0036.sql

```diff
@@ -0,0 +1,389 @@
+-- May 31, 2016
+-- Add surfer and fermeted foods questionnaires
+
+----------------------------------------------------------
+-- survey_group
+----------------------------------------------------------
+INSERT INTO survey_group (group_order, american, british) VALUES (-3, 'Fermented Foods', 'Fermented Foods');
+INSERT INTO survey_group (group_order, american, british) VALUES (-4, 'Surfers', 'Surfers');
+
+----------------------------------------------------------
+-- surveys
+----------------------------------------------------------
+INSERT INTO surveys (survey_id, survey_group) VALUES (3, -3);
+INSERT INTO surveys (survey_id, survey_group) VALUES (4, -4);
+
+----------------------------------------------------------
+-- survey_question
+----------------------------------------------------------
+-- Fermented
+INSERT INTO survey_question (survey_question_id, question_shortname, american, british) VALUES
+(164, 'FERMENTED_FREQUENCY', 'How often do you consume one or more servings of fermented vegetables or plant products a day in an average week? (1 serving = ½ cup sauerkraut, kimchi or fermented vegetable or 1 cup of kombucha)', 'How often do you consume one or more servings of fermented vegetables or plant products a day in an average week? (1 serving = ½ cup sauerkraut, kimchi or fermented vegetable or 1 cup of kombucha)'),
```

Comment:
Yes, it has been tested and renders, but I will change to `1/2` just to ward off any future unexpected errors.